recipes/adfcosmorscompound.py

```diff
@@ -89,8 +94,9 @@ def __init__(
         coskf_dir: Optional[str] = None,
         preoptimization: Optional[str] = None,
         singlepoint: bool = False,
+        densf2hbc: bool = True,
```

Comment:
(technically breaking if people are using positional arguments, but probably can get away with it)